content/docs/static-analysis/codeql/_index.md

```diff
@@ -14,6 +14,11 @@ support for both inter- and intraprocedural control flow and data flow analysis.
 However, be aware that the learning curve for writing your own custom queries
 is steep, and documentation for the CodeQL standard libraries is still scant.
 
+{{< hint info >}}🎥 Watch the Trail of Bits Webinar on
+[Introduction to CodeQL: Examples, Tools and CI Integration](https://www.youtube.com/watch?v=rQRlnUQPXDw),
+where we lead a session on how to get started with CodeQL, as part of the second chapter
```

Comment:
I think we can remove the "as part of the second chapter ..." as it does not add important information (and the codeql chapter is even displayed first on the page =) 

Something like `Watch the Trail of Bits Webinar on
[Introduction to CodeQL: Examples, Tools and CI Integration](https://www.youtube.com/watch?v=rQRlnUQPXDw),
where we show you how we used CodeQL to find real-world security issues, the tools needed for an effective CodeQL experience, and how to set up your CodeQL CI integration`

Reply:
https://github.com/trailofbits/testing-handbook/pull/24/commits/a1624993552c140162c6b58d88e6ae65cd5f7b6c